### ISSUE TITLE
Touchscreen delta insane value [2019.1.13f1 - 0.9.2]

### ISSUE THREAD

**Issue by newAnim** [Unity-Technologies/InputSystem#785]
Using mouse delta does give me normal values, but after a build on android ; i get a million negative value from primary touch / touch0 delta

**Comment by Rene-Damm** [comment]
Is this in the player or in the input debugger?

**Comment by newAnim** [comment]
It was in the player, I found a workaround by using the UI and IPointerData ; tutorials helped a lot

**Comment by Rene-Damm** [comment]
I see. Glad you found a workaround.

To make sure we check this on our side, I've migrated the issue to a FogBugz ticket as part of migrating all our bug tracking to the official Unity bug tracking system (which gives us the benefit of properly integrating ourselves with Unity QA procedures) and turn off GitHub issue tracking once done.

FB Ticket: 1190517
Unity Issue Tracker: https://issuetracker.unity3d.com/product/unity/issues/guid/1190517/

Closing the issue here. No worries, doesn't mean it's forgotten :)
